src/Project/Upload.py

```diff
@@ -55,7 +55,9 @@ def upload_project(self):
             self.entry.data["hash"] = Hash.get_project_hash(self)
             self.entry.update()
 
-            Slack(f'{Slack.get_nice_username()} uploaded a new version of {Slack.make_nice_project_name(self.entry.name)}')
+            content = f"{Discord.get_nice_username()} uploaded a new version " \
+                    "of {Discord.make_nice_project_name(self.entry.name)}"
```

Comment:
Actually same here with make_nice_project_name.  I made a comment about this in 'menu_create_project.py'